koshmar/src/main/java/org/testmonkeys/koshmar/core/factory/PageScanner.java

```diff
@@ -38,4 +49,18 @@ private Supplier pageWithNameNotFound(String name) {
         Optional<Class<?>> first = typesAnnotatedWith.stream().filter(isPageWithName(name)).findFirst();
         return first.orElseThrow(pageWithNameNotFound(name));
     }
+
+    public <T extends AbstractComponent> T findPageElementByName(Page page, String name) {
```

Comment:
The class is mostly used to scan the packages for page objects classes, method, by it's name seems to need a separate class